### PR TITLE
Fix vulnerabilities for AzureSpringCloudV0 task

### PR DESCRIPTION
**Pull request by KonstantinTyukalov**
**Task name**: AzureSpringCloudV0

**Description**: Resolved critical vulnerabilities by running the 'npm audit fix' command

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Comment by KonstantinTyukalov** [comment]
@yevster could you take a look?

File: Tasks/AzureSpringCloudV0/task.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "preview": true,
     "version": {
         "Major": 0,
-        "Minor": 201,
+        "Minor": 205,
```
Comment (sergei-fedorov) [inline Tasks/AzureSpringCloudV0/task.json:21]: Shouldn't it be `204`?

**Review by anatolybolshakov — APPROVED** [review]

**Comment by DenisRumyantsev** [comment]
Hi, @lucen-ms, could you take a look at this PR and test these changes?

**Comment by DenisRumyantsev** [comment]
The `AzureSpringCloudV0` task does not have an owner.
This task should be assigned to some team before work on this PR will continue.

**Comment by KonstantinTyukalov** [comment]
Actual PR - https://github.com/microsoft/azure-pipelines-tasks/pull/16602

**Pull request closed without merging**